### DISCUSSION TITLE
NER - Finding entity's character positions

### DISCUSSION THREAD

**Discussion by mattkallo** [explosion/spaCy#12942] [Help: Coding & Implementations] [answered]
How can we get individual entity's starting and ending char positions?  doc.ents "start" and "end" gives the token positions. 

```
print([(ent.text, ent.label_, ent.start, ent.end) for ent in doc.ents])
[('Amazon', 'ORG', 0, 1), ('Rxpass', 'PRODUCT', 2, 3)........

```

Thanks

**Comment by mattkallo** [comment, marked as answer]
Found the answer, incase anyone looking for same info.
  
`print(ent.text, ent.start_char, ent.end_char)`

**Reply by svlandeg** [reply]
Thanks for posting back with the solution. Just FYI - you should typically be able to find this kind of information in the "API" section of the docs, in this case over at https://spacy.io/api/span#attributes.
